packages/node-core/src/indexer/sandbox.ts

```diff
@@ -155,3 +155,27 @@ export class IndexerSandbox extends Sandbox {
     this.freeze(logger, 'logger');
   }
 }
+
+export class TestSandbox extends Sandbox {
```

Comment:
Do you think we need this extra class? Can we just use the IndexerSandbox class?

We're missing injecting the api here too.

Reply:
this sandbox is only used for loading the tests. The tests are later run in the IndexerSandbox with the injected globals.